src/transformers/training_args.py

```diff
@@ -482,6 +482,11 @@ class TrainingArguments:
                 - sync_module_states (`bool`, *optional*, defaults to `True`)
                     If `"True"`, each individually wrapped FSDP unit will broadcast module parameters from rank 0 to
                     ensure they are the same across all ranks after initialization
+                - cpu_ram_efficient_loading (`bool`, *optional*, defaults to `False`)
+                    If `"True"`, only the first process loads the pretrained model checkpoint while all other processes
+                    have empty weights.  When this setting is True, `sync_module_states` also must to be True,
+                    otherwise all the processes except the main process would have random weights leading to unexpected
+                    behaviour during training.
```

Comment:
For consistency 
```suggestion
                    If `True`, only the first process loads the pretrained model checkpoint while all other processes
                    have empty weights.  When this setting is `True`, `sync_module_states` also must to be `True`,
                    otherwise all the processes except the main process would have random weights leading to unexpected
                    behaviour during training.
```